master/buildbot/db/builds.py

```diff
@@ -302,5 +302,6 @@ def _minimal_bdict(row, botmaster):
             project=botmaster.getBuilderConfig(row.buildrequests_buildername).project,
             slavename=row.builds_slavename,
             submitted_at=mkdt(row.buildrequests_submitted_at),
-            complete_at=mkdt(row.buildrequests_complete_at)
+            complete_at=mkdt(row.buildrequests_complete_at),
+            sourcestampsetid=row.buildsets_sourcestampsetid,
```

Comment:
Are you sure we need it as minimal information? 
I think the sourcestampsetid is required for only one method, so maybe we should add this information only there.

Reply:
`_minimal_bdict' is used only by `getLastBuildsOwnedBy` and I just added the line into existing code. I agree, that method is ugly because: 
- is staticmethod
- is unnecessary